algorithms/src/crh/bhp.rs

```diff
@@ -86,27 +86,25 @@ impl<G: ProjectiveCurve, const NUM_WINDOWS: usize, const WINDOW_SIZE: usize> CRH
         }
     }
 
-    fn hash(&self, input: &[u8]) -> Result<Self::Output, CRHError> {
+    fn hash_bits(&self, input: &[bool]) -> Result<Self::Output, CRHError> {
```

Comment:
Now all the CRH implements the `hash_bits`, and the original `hash` will be mapped to `hash_bits`, by the trait definition.